frontend/src/metabase/archive/containers/ArchiveApp.tsx

```diff
@@ -53,16 +53,27 @@ export function ArchiveApp() {
   const { clear, getIsSelected, selected, selectOnlyTheseItems, toggleItem } =
     useListSelect<CollectionItem>(item => `${item.model}:${item.id}`);
 
-  const list = useMemo(() => {
-    clear();
-    return data ?? [];
-  }, [data, clear]);
+  const writableList = useMemo(() => {
+    clear(); // clear selected items if data is ever refreshed
+
+    if (!Array.isArray(data)) {
+      return [];
+    }
+
+    if (isAdmin) {
+      return data;
+    }
```

Comment:
I'm on the fence about this, but I think we should delete this check. With Auditv2 we've had to remove and change a lot of logic that assumes that admins can edit anything since that's no longer true. In its current form, Audit collections shouldn't be archive-able, but I'd like to move to a world where we just rely on `can_write` instead of assuming that admins can edit anything.